packages/smart-contracts/scripts/deploy-payments.ts

```diff
@@ -38,7 +37,7 @@ import { RequestLogicTypes } from '@requestnetwork/types';
  */
 export async function deployAllPaymentContracts(
   args: any,
-  hre: HardhatRuntimeEnvironment,
+  hre: HardhatRuntimeEnvironmentExtended,
```

Comment:
This way we can compute the address of a contract deployed via the RequestDeployer